content/en/docs/tasks/observability/distributed-tracing/skywalking/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Apache SkyWalking
+description: Learn how to configure the proxies to send tracing requests to Apache SkyWalking.
+weight: 10
+keywords: [telemetry,tracing,skywalking,span,port-forwarding]
+aliases:
+    - /docs/tasks/skywalking-tracing.html
```

Comment:
I'm not sure why this alias is here. Usually only needed when changing the location of a document.

Reply:
Removed